blog/2021-03-16-BatchSolvingWithAmq.adoc

```diff
@@ -0,0 +1,182 @@
+= Batch solving in a stateless manner with OptaPlanner and AMQ
+:page-interpolate: true
+:imagesdir: 2021/03/16
+:awestruct-author: rsynek
+:awestruct-layout: blogPostBase
+:awestruct-tags: [amq, quarkus, messaging, integration]
+:awestruct-share_image_filename: architecture.png
+
+Let's take a look at a new https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/amq-quarkus-school-timetabling[quickstart], which shows how to build a scalable, stateless service capable of solving numerous instances of the same planning problem.
+This post will be most interesting to developers looking for how to bring OptaPlanner from an embedded engine to a service communicating with other applications.
+
+== Batch Solving
+
+Very often, there are multiple instances of the same planning problem to solve.
+Either these come from splitting enormous input problem into smaller pieces, or just from the need to solve completely unrelated data sets.
+Imagine scheduling routes of vehicles independently per regions, or optimizing school timetables for numerous schools.
+To take advantage of time, you run OptaPlanner every night to prepare for the next day in the business or even longer for the next semester.
+On the other hand, during the day or in the middle of the semester, there is nothing to optimize and so there should be no OptaPlanner running.
```

Comment:
This isn't clear to me. I understand why I wouldn't run OptaPlanner after the schedule starts but how does this relate to batch solving?

Reply:
The batch solving doesn't only mean there are multiple input problems, but also there is a quiet period between solving such groups of problems. At least I see this meaning. Would you recommend rephrasing that part of the paragraph?